modules/types/gamedef/meta.ts

```diff
@@ -13,6 +13,7 @@ export type AlgolMeta<Blob extends AlgolGameBlobAnon> = {
   slug: string;
   demo?: string;
   added: string;
+  addedBy: string;
```

Comment:
Sorry, I forgot - since the other games don't have this, we have to make it optional for now!

```suggestion
  addedBy?: string;
```

Reply:
Ah, I should have spotted that as well. :D